src/test/visit_test_suite.py

```diff
@@ -1013,18 +1056,9 @@ def main(opts,tests):
         ridx  = pjoin(opts["result_dir"],"results.json")
         tests = load_test_cases_from_index(opts["tests_dir"],ridx,True)
     elif len(tests) == 0:
-        tests = find_test_cases(opts["tests_dir"],opts["classes"])
-    tests = [ abs_path(pjoin(opts["tests_dir"], "..",t)) for t in tests]
-    # use glob to match any *.py
-    expandedtests = []
-    for t in tests:
-       if not '*' in t:
-          expandedtests.append(t)
-       else:
-          for match in glob.iglob(t):
-             expandedtests.append(match)
-    if len(expandedtests) > 0:
-        tests = expandedtests
+        tests = find_test_cases(opts["tests_dir"],opts["classes"]) 
+    # reckon test paths using helper
+    tests = resolve_test_paths(tests,opts["tests_dir"])
```

Comment:
@biagas I refactored this logic into a function, when you get a chance can you take a quick look to see if it still captures your changes.  I think I need to drop the  

`if sys.platform.startswith("win"):` ?

- [ ]  have kathleen double check

Reply:
@biagas can you take a quick look at this to see if removing `if sys.platform.startswith("win"):` gets us back to your solution?

Reply:
Yes, I think it will.

Reply:
thanks, I'll make that change!